Exiled.Events/EventArgs/Map/ChangedIntoGrenadeEventArgs.cs

```diff
@@ -15,7 +15,7 @@ namespace Exiled.Events.EventArgs.Map
     using InventorySystem.Items.ThrowableProjectiles;
 
     /// <summary>
-    /// Contains all information for when the server is turned a pickup into a live grenade.
+    /// Contains all information after a pickup grenade was changed into a live grenade.
```

Comment:
Use <see cref="GrenadePickup"/> instead of pickup grenade
Same for live grenade